src/plugins/compare/file_coverage/code/file_coverage.py

```diff
@@ -164,6 +171,23 @@ def _evaluate_entropy_for_list_of_uids(self, list_of_uids, new_result, firmware_
         if non_zero_file_ids:
             new_result[firmware_uid] = non_zero_file_ids
 
+    # diverging text files
```

Comment:
This comment is redundant with the function name.
Maybe the function should have a docstring.

Reply:
removed the comment and added a docstring